Packs/PaloAltoNetworks_PAN_OS_EDL_Management/Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.py

```diff
@@ -620,8 +620,6 @@ def edl_get_external_file_metadata_command(args: dict):
 
     file_size = int(result.split("Size: ", 1)[1].split(" ", 1)[0])
     file_name = file_path.split("/")[-1]
-    if len(file_name) < 0:
-        file_name = file_path
```

Comment:
Hi @fabihatasneem Thanks for your contribution, just a quick question. What is this change here supposed to address?

Reply:
It's a logical expression which was inappropriately used, because the length of a collection is always greater than or equal to zero. So testing that a length is less than zero is always false, hence the code block is unreachable.